src/Analyzers/CSharp/Analyzers/AddRequiredParentheses/CSharpAddRequiredPatternParenthesesDiagnosticAnalyzer.cs

```diff
@@ -42,7 +42,7 @@ protected override bool IsBinaryLike(PatternSyntax node)
         protected override (PatternSyntax, SyntaxToken, PatternSyntax) GetPartsOfBinaryLike(BinaryPatternSyntax binaryPattern)
         {
             Debug.Assert(IsBinaryLike(binaryPattern));
-            return (binaryPattern.LeftPattern, binaryPattern.PatternOperator, binaryPattern.RightPattern);
+            return (binaryPattern.LeftPattern, binaryPattern.OperatorToken, binaryPattern.RightPattern);
```

Comment:
didn't we want LeftPattern and RightPattern to be renamed to Left/Right to also match BinaryExpression?

Reply:
> didn't we want LeftPattern and RightPattern to be renamed to Left/Right to also match BinaryExpression?

This is the first time I'm aware of that suggestion. @dotnet/roslyn-compiler Any opinions?

Reply:
Sorry, i thought it was clear in https://github.com/dotnet/roslyn/issues/43452  :)

Yes, i'm asking for BinaryPattern/Expr to be uniform here in naming for all the children.  Thanks!

Reply:
I also assumed this is part of the proposal in #43452 and commented there with that in mind.